tests/foreman/sys/test_rename.py

```diff
@@ -0,0 +1,128 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-change-hostname``
+
+:Requirement: katello-change-hostname
+
+:CaseAutomation: notautomated
+
+:CaseLevel: System
+
+:CaseComponent: katello-change-hostname
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+from fauxfactory import gen_string
+from robottelo.config import settings
+from robottelo.decorators import (
+        destructive,
+        stubbed,
+)
+from robottelo.ssh import get_connection
+from robottelo.test import TestCase
+
+BCK_MSG = "Hostname change complete!"
+
+
+@destructive
+class RenameTestCase(TestCase):
+    """Implements ``katello-change-hostname`` tests"""
+
+    @stubbed
+    def test_positive_rename_satellite(self):
+        """run katello-change-hostname on Satellite server
+
+        :id: 9944bfb1-1440-4820-ada8-2e219f09c0be
+
+        :setup: Satellite server with synchronized rh and custom
+            repos and with a registered host
+
+        :steps:
+            1. Rename Satellite using katello-change-hostname
+            2. Do basic checks for hostname change (hostnamctl)
+            3. Run some existence tests, as in backup testing
+            4. Verify certificates were properly recreated, check
+                for instances of old hostname
+                in etc/foreman-installer/scenarios.d/
+            5. Check for updated repo urls, installation media paths,
+                updated internal capsule
+            6. Check usability of entities created before rename: refresh
+                manifest, resync repos, republish CVs and re-register hosts
+            7. Create new entities (run end-to-end test from robottelo)
```

Comment:
Also the setup , part is not implemented. 
Cannot see these checks in the automated code? 3,4,5,6,7
Please correct me , if i am missing something